src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -311,6 +352,10 @@ public override void RequestRefresh()
         {
             DateTimeOffset now = DateTimeOffset.UtcNow;
 
+            _telemetryClient.IncrementConfigurationRefreshRequestCounter(
+                MetadataAddress,
+                TelemetryConstants.Protocols.Direct);
```

Comment:
Using the name Protocols.Direct in a method called RequestRefresh is kind of confusing.